src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const notificationTitle = document.createElement('h2');
+  const notificationDescription = document.createElement('p');
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
+
+  notification.style.top = posTop + 'px';
```

Comment:
Use string interpolation for readability
```suggestion
  notification.style.top = `${posTop}px`;
```